src/main/java/ch/njol/skript/util/Utils.java

```diff
@@ -65,6 +64,8 @@
 public abstract class Utils {
 	
 	private Utils() {}
+
+	private static final BiMap<EntityData, EntityType> spawnerTypes = HashBiMap.create();
```

Comment:
Does this not refer to Skript's EntityType, while its values are Bukkit's EntityType?

Reply:
If not, the full class reference in the new methods isn't needed

Reply:
It is indeed, it was changed by accident when i copy-pasted